website/js/DataTable.js

```diff
@@ -36,8 +36,24 @@ function setPages({data, count, deletedCount, synonyms}, pageLength) { //eslint-
 // Wrap Table with a version having PureRenderMixin
 var FastTable = React.createClass({
     mixins: [PureRenderMixin],
+    truncateGenomicCoordinates: function(variantData) {
```

Comment:
unfortunately, CSS is not an option for selectively truncating fields with the current system we use to populate our search table.